include/z64.h

```diff
@@ -1498,7 +1498,7 @@ typedef struct struct_8013DF3C_arg1 {
 typedef struct {
     /* 0x00 */ u32 type;
     /* 0x04 */ u32 setScissor;
-    /* 0x08 */ Color_RGBA8 color;
+    /* 0x08 */ Color_RGBA8_u32 color;
     /* 0x0C */ Color_RGBA8 envColor;
 } struct_801F8010; // size = 0x10
```

Comment:
What is this struct doing? Is this limited to `z_vimode`? Or is this related to other vi modes.